lib/redis/pipeline.rb

```diff
@@ -122,7 +122,7 @@ def _set(object)
     end
 
     def _command
-      @command
+      @command.map(&:to_s)
```

Comment:
This is a bit clever I recognize, the idea it to cast all the commands to string eagerly so that `call_pipelined` is sure not to trigger any side effect.